web-registry/src/generated/json-ld/project-metadata.ts

```diff
@@ -16,6 +16,8 @@ export interface ProjectMetadataLD {
   'regen:landOwner': ProjectStakeholder;
   'regen:projectOriginator': ProjectStakeholder;
   'regen:offsetGenerationMethod': string;
+  // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
+  'http://regen.network/offsetGenerationMethod'?: string;
```

Comment:
@wgwz Not sure why I couldn't get this to compact as expected.

Reply:
Glad to take a look at this with you another time if it's not blocking us at the moment. But if you do want to spend some time on it, just let me know